### PR TITLE
Fixing the documentation on how to correctly copy the fieldsets with custom attributes

### PR DESCRIPTION
**Pull request by eduard13**
## This PR is a:

- Content update

## Summary

When this pull request is merged, it will improve the documentation on how to copy correctly the fieldset from quote to order.

This PR fixes the #2910 issue.

## Additional information

List all affected URLs 

- https://devdocs.magento.com/guides/v2.1/ext-best-practices/tutorials/copy-fieldsets.html
- https://devdocs.magento.com/guides/v2.2/ext-best-practices/tutorials/copy-fieldsets.html
- https://devdocs.magento.com/guides/v2.3/ext-best-practices/tutorials/copy-fieldsets.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.0/ext-best-practices/tutorials/copy-fieldsets.md [hunk 1]
```diff
@@ -53,45 +53,61 @@ The code snippet in the next step uses the name of the fieldset and aspect to sp
 ## Step 3: Copy the fieldset
 {:#step-3}
 
+For copying the fieldset, we'll observe the `sales_model_service_quote_submit_before` event by using the following code in our `events.xml` file:
+
+{% highlight xml %}
```
Comment (osrecio) [inline guides/v2.0/ext-best-practices/tutorials/copy-fieldsets.md]: Can you change it by ``` xml

File: guides/v2.1/ext-best-practices/tutorials/copy-fieldsets.md [hunk 2]
```diff
@@ -53,46 +53,65 @@ The code snippet in the next step uses the name of the fieldset and aspect to sp
 ## Step 3: Copy the fieldset
 {:#step-3}
 
+For copying the fieldset, we'll observe the `sales_model_service_quote_submit_before` event by using the following code in our `etc/events.xml`:
+
+{% highlight xml %}
```
Comment (osrecio) [inline guides/v2.1/ext-best-practices/tutorials/copy-fieldsets.md]: Can you change it by ``` xml

Comment (eduard13) [reply guides/v2.1/ext-best-practices/tutorials/copy-fieldsets.md]: @osrecio thank you for review. I've adjusted the xml snippets, but if I'm not mistaken, there shouldn't be that space, correct me if I'm wrong.

Comment (osrecio) [reply guides/v2.1/ext-best-practices/tutorials/copy-fieldsets.md]: You're right, without spaces, I edited my comment before but you just write your comment hehe. 

Thanks!

**Comment by dobooth** [comment]
Hi @eduard13 
Thanks for the contribution.
We aren't making substantive changes to the v2.0 docs, just typos and tiny required changes.
Please back out the v2.0 file from this PR and then we can get it merged.
Thanks!

**Comment by eduard13** [comment]
Hi @dobooth, I've reverted those changes.
Thank you for feedback.

**Comment by eduard13** [comment]
Hey @dobooth, any updates on this one?
Unfortunately I'm not able to remove the label.

**Comment by dobooth** [comment]
Sorry about that!

**Comment by erikmarr** [comment]
running tests

**Comment by erikmarr** [comment]
running tests

**Comment by erikmarr** [comment]
Thanks @eduard13!

**Pull request merged**